### DISCUSSION TITLE
Why I can't save my proposal ?

### DISCUSSION THREAD

**Discussion by metrox-eth** [snapshot-labs/snapshot#765] [Q&A] [answered]
Suggest answers below or vote for an answer.

**Comment by metrox-eth** [comment, marked as answer]
- Make sure you filled "Question" and "What is you proposal" placeholders.
- Make sure you've added at least two choices.
- Make sure you've selected a set of actions for voting system, start and end date.
- Make sure to enter a "snapshot block number"

If you still cant's save your proposal contact us on Discord or Telegram

**Reply by abdelshok** [reply]
What is the "snapshot block number"? How do you decide on it or where do you find it?

**Reply by metrox-eth** [reply]
The snapshot block number is the block we read the voting power from. It's like a still picture of the blockchain in moment in time. When you create a proposal the last block is prefilled.
